airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/DocumentationPanelContext.tsx

```diff
@@ -29,13 +29,6 @@ export const useDocumentationPanelState = () => {
 // @ts-expect-error Default value provided at implementation
 export const documentationPanelContext = createContext<DocumentationPanelContext>();
 
-export const useCloseDocumentationPanelEffect = () => {
-  const { setDocumentationPanelOpen } = useDocumentationPanelState();
-  useEffect(() => {
-    return setDocumentationPanelOpen(false);
-  }, [setDocumentationPanelOpen]);
-};
-
```

Comment:
I see this pattern repeated in many places and I wonder if this hook was meant to be used but never happened.

Reply:
The pattern of closing hooks or the pattern of having code introduced that we never used?

I think once we fully cleaned up dead code we might want to introduce some checks into the CI that at least warn about having unused code around.

Reply:
I meant that if you search for `setDocumentationPanelOpen(false)` in the code, you'll see that this snipppet is repeated in multiple places:

```ts
  const { setDocumentationPanelOpen } = useDocumentationPanelState();
  useEffect(() => {
    return setDocumentationPanelOpen(false);
  }, [setDocumentationPanelOpen]);
};
```

Reply:
Yeah I agree we could either improve this by making the context methods maybe a `openDocumentationPanel` and `closeDocumentationPanel` or really a hook that auto closes it on unmount. I'd like do that refactoring though not in this PR. But I am fine removing the removal of this code from the PR again, if we want to keep it around for the refactoring to happen.